dsl_parser/elements/operation.py

```diff
@@ -415,3 +428,27 @@ def process_operation(
                     'workflow' if is_workflows else 'operation'))
         error_message = base_error_message + partial_error_message
         raise exceptions.DSLParsingLogicException(error_code, error_message)
+
+
+def _is_explicit_script(script):
```

Comment:
Leaving aside the implementation of this (which I'm still considering and we're also discussing out of band), I'd probably call this _is_inline_script